### ISSUE TITLE
Output text trashed after using latest godot alpha

### ISSUE THREAD

**Issue by xX-Void-Xx** [godotengine/godot#64955]
### Godot version

Godot_v3.5-stable_win64 [After Using]-> Godot_v4.0-alpha14_win64

### System information

Windows 7 64bit

### Issue description

while using Godot v3.5 the output box was working just fine before, all the text was showing in plain english untill I decided to try out the latest godot alpha. the new version worked just fine (albeit a bit laggy), after a bit of playing arround I went back to working on my game in Godot 3.5 just to find this:
![Capture0](https://user-images.githubusercontent.com/69814029/187025806-a71cf4da-d521-4374-979f-bbc9becb4312.PNG)

it looks like a text rendering problem ? I don't really know
note that the output window is the only one with this weird text, all other text in the debugger and everywhere else works just fine

### Steps to reproduce

Open [Godot_v4.0-alpha14_win64 ](https://downloads.tuxfamily.org/godotengine/4.0/alpha14/), play arround for a minute, leave and then open Godot_v3.5-stable_win64
(not sure if this works for other earlier versions)

**Comment by akien-mga** [comment]
Does it work fine if you restart Godot, or reboot the computer? The graphics drivers might be in a broken state.

**Comment by xX-Void-Xx** [comment]
rebooting the computer does solve this apparently, I was so frustrated trying to find a solution that I didn't think of the good old reboot method. my apologies, I'll close the issue now
